manifests/overlays/standalone/kustomization.yaml

```diff
@@ -2,8 +2,9 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: kubeflow
 resources:
-  - ../../base
-  - namespace.yaml
+- ../../base
+- namespace.yaml
 images:
-  - name: kubeflow/training-operator
-    newTag: "v1-855e096"
+- name: kubeflow/training-operator
+  newName: quay.io/opendatahub/training-operator
```

Comment:
Could this be patched from the `rhoai` overlay?

Reply:
sure, I'Il update the PR accordingly